packages/intent/src/cli.ts

```
@@ -10,9 +10,16 @@ import {
   scanIntentsOrFail,
 } from './cli-support.js'
 import { runEditPackageJsonCommand } from './commands/edit-package-json.js'
-import { runInstallCommand } from './commands/install.js'
-import { runListCommand } from './commands/list.js'
+import {
+  runInstallCommand,
+  type InstallCommandOptions,
```

ESLint

[error] Member 'InstallCommandOptions' of the import declaration should be sorted alphabetically.

(sort-imports)